lib/ansible_tower_client/base_model.rb

```diff
@@ -157,6 +157,10 @@ def hashify(attribute)
       YAML.safe_load(send(attribute))
     end
 
+    def reload
```

Comment:
@bdunne - The call works, but using the following test - I'm not getting the caller updated as I would have expected:

```
project = conn.api.projects.all.first
2.3.0 :008 > project.description
 => "A great demo"
2.3.0 :009 > project.description = "blah"
 => "blah"
2.3.0 :011 > project.reload
<output of the called object>
2.3.0 :013 > project.description
 => "blah"
```
I would have expected `project.description` to equal 'A great demo'

I had that same issue with `update_attributes` and had to re call each method to update them [inline](https://github.com/ansible/ansible_tower_client_ruby/blob/master/lib/ansible_tower_client/base_model.rb#L84-#L89)